htdocs/blockedlog/ajax/authority.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
For first version of module, we don't need external authority. We can imagine to support this in future, but for certification, this is no required and standard version should work without any external access.